contracts/PixelmonPartySquad/PxPartySquad.sol

```diff
@@ -0,0 +1,392 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.16;
+
+import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "./PsWeekManager.sol";
+import "./IPsChainlinkManager.sol";
+
+/// @notice Thrown when all treasures are already claimed
+error AlreadyClaimed();
+/// @notice Thrown when address is not a winner
+error NotAWinner();
+/// @notice Thrown when input is not as expected condition
+error InvalidInput();
+/// @notice Thrown when treasure index doesn't exist
+error InvalidTreasureIndex();
+/// @notice Thrown when no available treasures to be transferred to the winner
+error InsufficientToken();
+/// @notice Thrown when the input signature is invalid.
+error InvalidSignature();
+
+contract PxPartySquad is PsWeekManager, ReentrancyGuard {
+    /// @notice code number for ERC1155 token
+    uint8 public constant ERC_1155_TYPE = 1;
+    /// @notice code number for ERC721 token
+    uint8 public constant ERC_721_TYPE = 2;
+
+    /// @notice Wallet address that keeps all treasures
+    address public vaultWalletAddress;
+
+    uint256 public maxSpecialTreasureLimit = 1;
+
+    
+    /// @notice Variable to store Special Treasures treasure information such
+    ///         as the collection address, token ID, amount, and token type
+    Treasure public specialTreasure;
+    /// @notice List of addresses who have won Special Treasures 
+    /// @custom:key wallet address
+    /// @custom:value 'true' means already owns Special Treasures
```

Comment:
I think the comment needs to be rephrased.
the mapping value has uint256 type but the comment says _"@custom:value 'true' means already owns Special Treasures"_
Could you clarify it means and how we suppose to set the value?